prosody/rootfs/defaults/conf.d/jitsi-meet.cfg.lua

```diff
@@ -41,14 +50,22 @@ VirtualHost "{{ .Env.XMPP_DOMAIN }}"
     authentication = "internal_hashed"
   {{ end }}
 {{ else }}
-    authentication = "anonymous"
+    -- https://github.com/jitsi/docker-jitsi-meet/pull/502#issuecomment-619146339
+    authentication = "token"
```

Comment:
IS this still the case @paweldomas ? Looks very counter-intuitive.

Reply:
It is the case because that token module extracts the resume token from the URL. Maybe this should be configured this way only for the WebSockets. I don't feel like writing another auth module - the default anonymous module would have to be modified I think.